### PR TITLE
feat: cms events media

### PR DESCRIPTION
**Pull request by simboonlong**
- https://www.pivotaltracker.com/story/show/177298545

### CODE REVIEW

File: public/admin/config.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 backend:
   name: git-gateway
-  branch: master
+  branch: feat/cms-events-media
+  # to change back to master once pr approved
```
Comment (cavacado) [inline public/admin/config.yml]: ⬆️  reminder

File: src/pages/media.tsx [hunk 2]
```diff
@@ -1,172 +1,21 @@
...
-      eventSlides:
-        "https://baft.org/docs/default-source/past-webinars/webinar-the-journey-to-paperless-trade.pdf?sfvrsn=6e9a20e1_2",
-    },
-  },
-];
+const getSortedByDateDesc = (items: any[]) => {
+  items.sort((a, b): number => {
+    return new Date(b.attributes.date).getTime() - new Date(a.attributes.date).getTime();
+  });
+  return items;
```
Comment (cavacado) [inline src/pages/media.tsx:15]: might want to use `compareDesc` from date-fns instead of rolling your own

https://date-fns.org/v1.28.0/docs/compareDesc

Comment (simboonlong) [reply src/pages/media.tsx:15]: ah ok

**Review by cavacado — APPROVED** [review]
🎉🎊

**Pull request merged**